<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ACNL Spotpass File Creator v1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="5">
     <widget class="QPushButton" name="BTN_HHS_Make">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Make House</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="LBL_MakeSPFiles">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Make Spotpass Files:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QPushButton" name="BTN_UnkSpotpass_Make">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Make Unknown</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="Line" name="LineH_1">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="6">
     <widget class="Line" name="LineH_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="LBL_SPFileSettings">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Spotpass File Settings:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QPushButton" name="BTN_Present">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Post Office Present</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QPushButton" name="BTN_RV">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Campground RV</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="BTN_RV_Make">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Make RV</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="QPushButton" name="BTN_Letter">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Letter Delivered by Pete</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="BTN_Present_Make">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Make Present</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QPushButton" name="BTN_HHS">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Happy Home Showcase House</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" rowspan="5">
     <widget class="Line" name="LineV_1">
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="BTN_Letter_Make">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Make Letter</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="4">
     <widget class="QPushButton" name="BTN_UnkSpotpass">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Unknown Spotpass</string>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QPushButton" name="BTN_About">
      <property name="text">
       <string>About ACNL SFC</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="Line" name="LineV_2">
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="LBL_BossRegion">
      <property name="text">
       <string>Spotpass Region:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="2">
     <widget class="QComboBox" name="CMB_Region">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Spotpass (BOSS) Files are region locked. &lt;/p&gt;&lt;p&gt;Use this to pick the region you would like the Spotpass File to be compatible with.&lt;/p&gt;&lt;p&gt;Note: &lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Orig X&lt;/span&gt; = ACNL Games that were released before the Welcome Amiibo Update and needed to download the update.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;WA X&lt;/span&gt; = ACNL Games that come with the Welcome Amiibo Update pre-Installed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">combobox-popup: 0;</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="maxVisibleItems">
       <number>8</number>
      </property>
      <property name="maxCount">
       <number>9</number>
      </property>
      <item>
       <property name="text">
        <string>Orig JPN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Orig USA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Orig EUR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Orig KOR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WA JPN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WA USA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WA EUR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WA KOR</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Boot9"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpen_Boot9">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open Boot9</string>
   </property>
   <property name="toolTip">
    <string>Open Boot9 to Extract AES Key from.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
